packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FLTGoogleMapJSONConversions.h

```diff
@@ -68,33 +92,13 @@ extern NSString *const kHeatmapGradientColorMapSizeKey;
 + (NSArray *)arrayFromLocation:(CLLocationCoordinate2D)location;
 + (UIColor *)colorFromRGBA:(NSNumber *)data;
 + (NSNumber *)RGBAFromColor:(UIColor *)color;
-+ (NSArray<CLLocation *> *)pointsFromLatLongs:(NSArray *)data;
-+ (NSArray<NSArray<CLLocation *> *> *)holesFromPointsArray:(NSArray *)data;
-+ (nullable GMSCameraPosition *)cameraPostionFromDictionary:(nullable NSDictionary *)channelValue;
-+ (GMSCoordinateBounds *)coordinateBoundsFromLatLongs:(NSArray *)latlongs;
-+ (nullable GMSCameraUpdate *)cameraUpdateFromArray:(NSArray *)channelValue;
```

Comment:
The reason these are moved instead of just renamed is that I'm chipping away at the functions-as-bag-of-static-methods-on-a-fake-class approach that the JSON conversion methods used, to the IMO-cleaner functions-as-functions approach. Once the conversion is complete this class will be gone.